diff --git a/pyproject.toml.orig b/pyproject.toml
index ae252156..3fbec9c4 100644
--- a/pyproject.toml.orig
+++ b/pyproject.toml
@@ -57,11 +57,9 @@ dependencies = [
   "requests>=2.26",
   "scipy>=1.10",
   "tqdm>=4.62.3",
-]
-
-optional-dependencies.metrics = [
   "pandas>=2",
 ]
+
 urls.Documentation = "https://supervision.roboflow.com/latest/"
 urls.Homepage = "https://github.com/roboflow/supervision"
 urls.Repository = "https://github.com/roboflow/supervision"
